### ISSUE TITLE
Add a Suggestion...

### ISSUE THREAD

**Issue by CheetahChrome** [NuGet/docs.microsoft.com-nuget#1288]
My issue with this warning was it started appearing after a failed attempt to install a Nuget Package with the wrong versions. 

Though the attempt did not change the project because it cleaned it self up, the warnings then remained. For me clearing the Nuget Cache and rebuilding worked. 

Could that be an appropriate alternate suggestion upon landing on this page? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0960d68b-f164-4e7e-5cdb-ad70e6f9de89
* Version Independent ID: 82b4d896-1d52-ce2e-c2ff-f76f4308ac56
* Content: [NuGet Warning NU1605](https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu1605?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3Den-US%26k%3Dk(NU1605)%26rd%3Dtrue)
* Content Source: [docs/reference/errors-and-warnings/NU1605.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/reference/errors-and-warnings/NU1605.md)
* Product: **nuget**
* GitHub Login: @zhili1208
* Microsoft Alias: **lzhi**

**Comment by BlueMarmalade** [comment]
Sometimes this error is because of upgrading from a .net core version to another(since some nuget libraries comes included by default that you previously had to add yourself). So for me, when upgrading from 2.1 to 2.2 I got ths error because my old entityframework nuget libraries are already included in the new version of .net core. So even if I upgraded the old ones, i got the error. I just needed to delete them without reinstalling them.
